docs/includes/file-get-started-excel-react.md

```diff
@@ -17,17 +11,21 @@ In this article, you'll walk through the process of building an Excel add-in usi
     npm install -g yo generator-office
     ```
 
-### Create the web app
+## Create the web app
 
 1. Create a folder on your local drive and name it **my-addin**. This is where you'll create the files for your app.
 
+    ```bash
+    mkdir my-addin
```

Comment:
Maybe this just clicked for me because I've seen several of these, but why do we create a folder when yo office also creates a folder?

Reply:
Excellent point! The "create a folder" step used to be necessary when Yo Office created all the new files in whatever directory it was run in. Now that Yo Office creates a folder to contain all the files it creates, this step isn't necessary.  I'll remove it from this quick start and all other quick starts in my next PR (coming later this afternoon).